release/models/system/openconfig-system-controlplane.yang

```diff
@@ -212,6 +242,66 @@ module openconfig-system-controlplane {
       }
     }
 
+    grouping system-control-plane-queue-config {
+      description
+        "Configuration data for queues associated with the
+        interface, this is re-used across input/output queues.";
+
+      leaf name {
```

Comment:
Should this be a leafref to the queue that is defined in the QoS model per the other approaches that we have in this model?

Reply:
> Couple of questions:
> 
> 1. This only seems (from the tree view) to do one queue -- don't we need this to be N queues?
> 2. Can we re-use the queue leaf definitions from the QoS model?  This seems to be a case of just re-using (`qos-interface-queue-state`)[https://github.com/openconfig/public/blob/master/release/models/qos/openconfig-qos-interfaces.yang#L318]

Done.

Also used leafref for the queue name.